src/components/Apps/AppInfo/AppInfo.tsx

```diff
@@ -31,25 +30,31 @@ export const AppInfo: FC<Props> = ({ app, onClose }) => {
       })
       .catch((_) => {
         // use fallback icon if image for id doesn't exist
-        // TODO: error handling
-        import("../../../assets/cloud.svg").then((img) =>
-          setIconImg(img.default)
-        );
+        import("../../../assets/cloud.svg")
+          .then((img) => setIconImg(img.default))
+          .catch((_) => {
+            // do nothing if cloud image doesn't exist either
+          });
       });
 
     // import app images
-    [1, 2, 3].forEach((num) => {
-      import(`../../../assets/apps/preview/${id}/${num}.png`)
-        .then((image) => {
+    Promise.allSettled([
```

Comment:
Nitpick: Is it possible to asign this to a var and use await? Like i.e. here: https://dmitripavlutin.com/promise-all-settled/#21-all-promises-fulfilled  
Might be easier to read.

Reply:
Okay, done :)